transaction/_manager.py

```diff
@@ -89,6 +89,8 @@ def registerSynch(self, synch):
         """ See ITransactionManager.
         """
         self._synchs.add(synch)
+        if self._txn is not None:
+            synch.newTransaction(self._txn)
```

Comment:
Should this behaviour be documented in ITransactionManager? ("If a synch is added while a transaction is in progress, it will immediately be notified via its `newTransaction` method.") The documentation to me doesn't seem to directly imply this.